pyppeteer/page.py

```diff
@@ -956,7 +956,7 @@ def set_request(req: Request) -> None:
             NetworkManager.Events.Response,
             lambda response: responses.__setitem__(response.url, response)
         )
-        result = await watcher.navigationPromise()
+        result = await watcher.navigationResponse()
```

Comment:
navigation response definitely isn't correct, but I think this can be resolved later on